### PR TITLE
scaffolds for product and showroom

### PR DESCRIPTION
**Pull request by tliber**

### CODE REVIEW

File: client/app/product/product.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="col-md-12">
+This is the product view.
```
Comment (kxprim) [inline client/app/product/product.html:2]: Indentation to correct

File: server/api/product/product.model.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
```
Comment (kxprim) [inline server/api/product/product.model.js:4]: We should follow our style guide by declaring each variable independently - no use of commas

File: server/api/showroom/showroom.model.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
```
Comment (kxprim) [inline server/api/showroom/showroom.model.js:4]: Variables should be defined with a 'var' each times

**Comment by kxprim** [comment]
Once changes are made it's all :+1:

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**